### DISCUSSION TITLE
Use mysql2 instead mysql

### DISCUSSION THREAD

**Discussion by murilomaiaa** [Daniel-Yonkov/jest-mysql#79] [Ideas]
I'm using mysql2 to connect to the database but this library requires mysql. I think it'll be fine to use mysql2 or mysql not just mysql

**Comment by Daniel-Yonkov** [comment]
@murilomaiaa  - Thanks for the idea! Would be happy to consider implementing and supporting multiple mysql wrapper libraries. Though it would be greatly appreciated if you provide a abstracted factory implementation where based on the user dependencies we can build their environment with the tooling of their choice.
Like the idea :)
